### DISCUSSION TITLE
Check marks don't appear next to an open Bunch

### DISCUSSION THREAD

**Discussion by bugdevil** [ttscoff/bunch-docs#314] [Q&A]
Hi,

When we toggle a bunch, a check mark should appear next to the open Bunch in the menu.

It doesn't work for me except for the "Example bunch", even though my "Toggle Bunches" option is enabled.

Is there a way to make it work plz?

(mac mini M1 2020, MacOS Sonoma 14.2.1)

**Comment by ttscoff** [comment]
I've never seen it not work. Which version of Bunch are you running?

**Reply by bugdevil** [reply]
It says I have the latest version : 1.4.16 (177)

**Reply by bugdevil** [reply]
I think the issue comes from Bartender. I closed Bartender and it worked. Would you have an idea on how to make them both work together?

**Reply by bugdevil** [reply]
Sorry, I'm not sure it comes from Bartender anymore.
Whether Bartender is opened or not:
- two bunches in a subfolder, including the Bunch Example, work properly (checkmark is visible).
- two bunches at the root don't work

**Comment by ttscoff** [comment]
Could you attach one of the non-working bunches, as well as a screenshot of your preferences, please.

Thanks,
Brett

**Comment by bugdevil** [comment]
you'll find the non-working bunch and the screenshot attached.

Thanks so much!
[Archive.zip](https://github.com/ttscoff/bunch/files/13853185/Archive.zip)
